src/api/Controllers/CoursesController.cs

```diff
@@ -33,36 +33,36 @@ public CoursesController(ICourseDbContext courseDbContext, ILogger<CoursesContro
             _logger = logger;
         }
 
-        [HttpPost("{providerCode}/{courseCode}")]
+        [HttpPut("")]
         [ApiTokenAuth]
-        public async Task<IActionResult> SaveCourse(string providerCode, string courseCode, [FromBody]Course course)
+        [RequestSizeLimit(100_000_000_000)]
+        public async Task<IActionResult> SaveCourses([FromBody]IList<Course> courses)
```

Comment:
You going to add the counterpart of this in the client?

Should there be additional validation to ensure that the list of courses are all for one provider?

Reply:
Hmm -- not sure S+C needs to enforce this as a restriction. It just happens to be how MC uses it